### PR TITLE
EJDOTNETCORE-4349 Need to add API references in PDF UG documentation

### PR DESCRIPTION
**Pull request by jeyalakshmit**
Hi Sir,

Added the API link and removed unwanted code spaces for below topics:
Digital Signature
Attachments
Security
supported-and-unsupported-features

Regards,
Jeyalakshmi T

### CODE REVIEW

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 1]
```diff
@@ -225,39 +197,32 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding a digital signature using stream
 
-The following code example illustrates how to add a digital signature in the PDF document
-using stream as follows.
+The following code example illustrates how to add a digital signature in the PDF document using stream and the [PdfSignature](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html) class as follows.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:200]: The following code example illustrates how to add a digital signature in the PDF document as stream using the [PdfSignature](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html) class as follows

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 2]
```diff
@@ -823,18 +710,16 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Signing an existing document
 
-You can load the signature field from the existing PDF document and add certificate to the document as follows.
+You can load the signature field from the existing PDF document and add certificate to the document by using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class as follows.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:713]: You can load the signature field from the existing PDF document using [PdfLoadedSignatureField]() class and add certificate to the document using [PdfCertificate]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 3]
```diff
@@ -987,32 +854,28 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Sign an existing document using stream
 
-You can load the signature field from an existing PDF document and add certificate to the document using stream as follows.
+You can load the signature field from an existing PDF document by using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class and add certificate to the document using stream as follows.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:857]: You can load the signature field from an existing PDF document using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class and add certificate to the document using stream as follows.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 4]
```diff
@@ -1158,22 +1000,20 @@ else
 
 ## Externally sign a PDF document 
 
-You can sign the PDF document from external digital signature created from various sources such as HSM, USB token, smart card, or other cloud services such as DigitalSign. The following code example shows how to sign the PDF document from external signature.
+You can sign the PDF document from external digital signature created from various sources such as HSM, USB token, smart card, or other cloud services such as DigitalSign. The following code example shows how to sign the PDF document from external signature using [ComputeHash](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_ComputeHash) Events.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:1003]: using [ComputeHash](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_ComputeHash) Event.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 5]
```diff
@@ -1366,27 +1199,23 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Create Long Term Validation (LTV) when signing PDF documents externally
 
-You can create a Long Term validation (LTV) when signing PDF documents externally using your private/public certificates.The following code example shows how to create an LTV when signing a PDF document from external signature.
+You can create a Long Term validation (LTV) when signing PDF documents externally using your private/public certificates.The following code example shows how to create an LTV when signing a PDF document from external signature using [CreateLongTermValidity](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_CreateLongTermValidity_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__) method.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:1202]: using [CreateLongTermValidity](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_CreateLongTermValidity_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__) method in [PdfSignature]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 6]
```diff
@@ -1545,8 +1355,7 @@ The PDF LTA signature is the next level of the LTV signature. It follows the sta
 
 The document timestamp is also applied to the PDF document, so it provides more viability to the signature. This level is recommended for qualified electronic signatures.
 
-The following code example shows how to sign a PDF document with LTA.
-
+The following code example shows how to sign a PDF document with LTA using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property and [EnableLtv](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_EnableLtv) property.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:1358]: The following code example shows how to sign a PDF document with LTA using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property and [EnableLtv](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_EnableLtv) property in [PdfSignature]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 7]
```diff
@@ -2109,45 +1881,39 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding a timestamp in digital signature
 
-Essential PDF allows you to add timestamp in the digital signature of the PDF document. The following code example explains the same.
+Essential PDF allows you to add timestamp in the digital signature of the PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property. The following code example explains the same.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:1884]: Essential PDF allows you to add timestamp in the digital signature of the PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property in [PdfSignature]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 8]
```diff
@@ -2344,21 +2081,19 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding a timestamp to PDF document
 
-You can add timestamp to the PDF document using the following code snippet.
+You can add timestamp to the PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property. The following code example explains the same.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:2084]: You can add timestamp to the PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property in [PdfSignature]() class. The following code example explains the same.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 9]
```diff
@@ -2485,28 +2204,25 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding a timestamp to existing PDF document
 
-You can add timestamp to the existing PDF document using the following code snippet.
+You can add timestamp to the existing PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property. The following code example explains the same.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:2207]: You can add timestamp to the existing PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property in [PdfSignature]() class. The following code example explains the same.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 10]
```diff
@@ -2649,40 +2348,33 @@ The Essential PDF provides support to get the certificate details from an existi
...
 
-You can get the above certificate details from an existing signed PDF document by using the following code snippet.
+You can get the above certificate details from an existing signed PDF document by using [SignedDate](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_SignedDate), [SignedName](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_SignedName), [SubjectName](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_SignedName), [IssuerName](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html#Syncfusion_Pdf_Security_PdfCertificate_IssuerName), [ValidFrom](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html#Syncfusion_Pdf_Security_PdfCertificate_ValidFrom) and [ValidTo](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html#Syncfusion_Pdf_Security_PdfCertificate_ValidTo) property.The following code example explains the same.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:2351]: Just add the link without (,)

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 11]
```diff
@@ -2786,46 +2465,40 @@ The Essential PDF supports creating long term signature validation when signing
 
 N> The resulted PDF document size will be large since all the necessary signature information, Certificate Revocation List (CRL), and Online Certificate Status Protocol (OCSP) are embedded.
 
-The following code example explains how to create LTV PDF.
+The following code example explains how to create LTV PDF using [PdfSignature](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html) class.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:2468]: he following code example explains how to create LTV using [EnableLtv]() property in [PdfSignature]() class when signing the PDF document.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 12]
```diff
@@ -3040,42 +2680,38 @@ The [PdfSignatureSettings](https://help.syncfusion.com/cr/file-formats/Syncfusio
 
 As per the PDF specification 2.0, now Syncfusion PDF library supports digital signature based on CAdES (CMS Advanced Electronics Signature). The CAdES based digital signature can remain valid for long periods, even if underlying cryptographic algorithms are broken. Using the API [CryptographicStandard](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_CryptographicStandard), you can change the standard between CMS (Cryptographic Message Syntax) and CAdES.
 
-The following code example explains how to add a digital signature with cryptographic standard (CAdES) to the PDF document.
+The following code example explains how to add a digital signature with [cryptographic standard](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_CryptographicStandard) (CAdES) to the PDF document.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:2683]: The following code example explains how to add a digital signature with [cryptographic standard](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_CryptographicStandard) as CAdES of [CryptographicStandard]() Enum when creating the PDF document.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 13]
```diff
@@ -3286,42 +2897,38 @@ The following message digest algorithms are now supported:
 * SHA512
 * RIPEMD160
 
-The following code example explains how to add a digital signature with various digest algorithms to the PDF document.
+The following code example explains how to add a digital signature with various digest algorithms to the PDF document using the [DigestAlgorithm](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_DigestAlgorithm) enum.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:2900]: The following code example explains how to add a digital signature with various digest algorithms to the PDF document by specifying the [DigestAlgorithm]() property as ``SHA256`` through [DigestAlgorithm](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_DigestAlgorithm) enum in [PdfSignatureSettings]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 14]
```diff
@@ -3932,12 +3457,12 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Deferred signing in PDF document
 
-The following code sample shows how to be deferred signing in a PDF document from an external signature.
+The following code sample shows how to be deferred signing in a PDF document from an external signature using [AddExternalSigner](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_AddExternalSigner_Syncfusion_Pdf_Security_IPdfExternalSigner_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__System_Byte___) method.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:3460]: The following code sample shows how to be deferred signing in a PDF document from an external signature using [IPdfExternalSigner]() class and [AddExternalSigner](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_AddExternalSigner_Syncfusion_Pdf_Security_IPdfExternalSigner_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__System_Byte___) method in[PdfSignature]().

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 15]
```diff
@@ -4666,13 +4119,12 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Deferred signing without PKCS7 encoding
 
-The following code sample shows deferred signing in a PDF document without PKCS7 encoding from an external signature.
+The following code sample shows deferred signing in a PDF document without PKCS7 encoding from an external signature using [AddExternalSigner](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_AddExternalSigner_Syncfusion_Pdf_Security_IPdfExternalSigner_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__System_Byte___) method.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:4122]: The following code sample shows deferred signing in a PDF document without PKCS7 encoding from an external signature using [IPdfExternalSigner]() class and [AddExternalSigner](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_AddExternalSigner_Syncfusion_Pdf_Security_IPdfExternalSigner_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__System_Byte___) method.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 16]
```diff
@@ -5418,33 +4792,28 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Long Term Validation (LTV) information
 
-Added support for LTV validation and getting CRL and OCSP embedded details from the digital signature.
+Added support for LTV validation and getting CRL and OCSP embedded details from the digital signature using [LtvVerificationInfo](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureValidationResult.html#Syncfusion_Pdf_Security_PdfSignatureValidationResult_LtvVerificationInfo) property.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:4795]: Added support for LTV validation and getting CRL and OCSP embedded details from the digital signature using [LtvVerificationInfo](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureValidationResult.html#Syncfusion_Pdf_Security_PdfSignatureValidationResult_LtvVerificationInfo) class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 17]
```diff
@@ -5700,7 +5032,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Remove existing digital signatures from a PDF document 
 
-The following code example illustrates how to remove existing digital signatures from a PDF document. 
+The following code example illustrates how to remove existing digital signatures from a PDF document using [Remove](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfFieldCollection.html#Syncfusion_Pdf_Interactive_PdfFieldCollection_Remove_Syncfusion_Pdf_Interactive_PdfField_) method.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:5035]: The following code example illustrates how to remove existing digital signatures from a PDF document using [Remove](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfFieldCollection.html#Syncfusion_Pdf_Interactive_PdfFieldCollection_Remove_Syncfusion_Pdf_Interactive_PdfField_) method in [PdfLoadedFormFieldCollection]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 18]
```diff
@@ -5820,7 +5149,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Certified signature 
 
-The following code snippet illustrates how to sign a PDF document without showing the digital signature.  
+The following code snippet illustrates how to sign a PDF document without showing the digital signature using [Certificated](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_Certificated) property.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:5152]: The following code snippet illustrates how to sign a PDF document without showing the digital signature using [Certificated](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_Certificated) property in [PdfSignature]() class.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 19]
```diff
@@ -6092,7 +5416,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding multiple signatures to a PDF document
 
-The following code example illustrates how to add multiple signatures to a PDF document without invalidating the previous signature.  
+The following code example illustrates how to add multiple signatures to a PDF document using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class without invalidating the previous signature.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:5419]: The following code example illustrates how to add multiple signatures to a PDF document without invalidating the previous signature.

Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:5419]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 18 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:5152]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 17 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:5035]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 16 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:4795]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 15 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:4122]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 14 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:3460]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 11 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:2468]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 9 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:2207]: Changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 10 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:2351]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 8 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:2084]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 7 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:1884]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 20]
```diff
@@ -1764,7 +1571,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 A Windows certificate store is a secure way to store the digital ID. If a root certificate is added to the Windows certificate store, you do not need to manually add and trust each of the certificates that are already present in the Windows certificate store.
 
-You can retrieve the digital ID "X509Certificate2" from the Windows certificate store and use it to add a digital signature to a PDF document.
+You can retrieve the digital ID [X509Certificate2](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html#Syncfusion_Pdf_Security_PdfCertificate__ctor_System_Security_Cryptography_X509Certificates_X509Certificate2_) from the Windows certificate store and use it to add a digital signature to a PDF document.
```
[the remark this answers is not in the document]
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:1574]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 6 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:1358]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 5 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:1202]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 4 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:1003]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 3 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:857]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 2 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:713]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 1 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:200]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 12 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:2683]: changed the content

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 13 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:2900]: Changed the content

**Comment by jeyalakshmit** [comment]
rebuild

**Review by praveenHari24 — APPROVED** [review]

**Pull request merged**